src/Translator.zig

```diff
@@ -2225,6 +2237,111 @@ fn transFloatLiteral(
     return t.maybeSuppressResult(used, as_node);
 }
 
+fn transDefaultInit(
+    t: *Translator,
+    scope: *Scope,
+    default_init: Node.DefaultInit,
+) TransError!ZigNode {
+    const init_type = try t.transType(scope, default_init.qt, default_init.last_tok);
+    return try ZigTag.std_mem_zeroes.create(t.arena, init_type);
+}
+
+fn transArrayInit(
+    t: *Translator,
+    scope: *Scope,
+    array_init: Node.ContainerInit,
+) TransError!ZigNode {
+    const array_item_type = try t.transType(scope, array_init.container_qt.childType(t.comp), array_init.l_brace_tok);
+    const maybe_array_filler: ?ZigNode = switch (array_init.items[array_init.items.len - 1].get(t.tree)) {
+        .array_filler_expr => |array_filler| blk: {
```

Comment:
Note that there can be multiple `array_filler_expr`s in an `array_init_expr`. It doesn't have to be implemented in this PR but it would be good to at least leave a note.

Reply:
Interesting, I'll just do that now.